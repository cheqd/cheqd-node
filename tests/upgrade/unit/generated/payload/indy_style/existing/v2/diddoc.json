{
    "did_doc": {
        "context": [
            "abc",
            "def"
        ],
        "id": "did:cheqd:testnet:xcWigjiDhDTaBGRX",
        "controller": [
            "did:cheqd:testnet:xcWigjiDhDTaBGRX"
        ],
        "verification_method": [
            {
                "id": "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-1",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:xcWigjiDhDTaBGRX",
                "verification_material": "{\"publicKeyMultibase\":\"z4xhmb8vKgCN24a3kRyai5F8eMSgkuqSA8ABPo8mYJSd\"}"
            },
            {
                "id": "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-2",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:xcWigjiDhDTaBGRX",
                "verification_material": "{\"publicKeyMultibase\":\"z6L84sd16WVYPQYQ6XXwdPtg2SEkxyj2CrsGGQ3MzkFP5\"}"
            },
            {
                "id": "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-3",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:xcWigjiDhDTaBGRX",
                "verification_material": "{\"publicKeyMultibase\":\"zADATkCngJDTbzxhzYtjC8ndzu5wWWbrH8UwQMowfcYDj\"}"
            },
            {
                "id": "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-4",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:xcWigjiDhDTaBGRX",
                "verification_material": "{\"publicKeyMultibase\":\"z5PuPVaqjigujtzrwQMZQviJV15gFSTVYkGEifGEGb6Ei\"}"
            }
        ],
        "authentication": [
            "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-1",
            "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-2"
        ],
        "assertion_method": [
            "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-3"
        ],
        "capability_invocation": [
            "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-4",
            "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-1"
        ],
        "capability_delegation": [
            "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-4",
            "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-2"
        ],
        "key_agreement": [
            "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-1",
            "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-2",
            "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-3",
            "did:cheqd:testnet:xcWigjiDhDTaBGRX#key-4"
        ],
        "also_known_as": [
            "alias-1",
            "alias-2"
        ],
        "service": [
            {
                "id": "did:cheqd:testnet:xcWigjiDhDTaBGRX#service-1",
                "serviceType": "type-1",
                "service_endpoint": [
                    "endpoint-1"
                ]
            }
        ]
    },
    "metadata": {
        "created": "2021-01-01T00:00:00Z",
        "updated": "2021-01-01T00:00:00Z",
        "deactivated": false,
        "version_id": "e604fc38-a7b3-4bf7-b0b0-053ad9b15957",
        "next_version_id": "",
        "previous_version_id": ""
    }
}