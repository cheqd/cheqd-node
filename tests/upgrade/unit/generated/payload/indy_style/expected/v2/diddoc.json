{
    "didDoc": {
        "context": [
            "abc",
            "def"
        ],
        "id": "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC",
        "controller": [
            "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC"
        ],
        "verificationMethod": [
            {
                "id": "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-1",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC",
                "verificationMaterial": "z6MkeXDkMqPMfDgq8ZQkRzwRZAo8TviYAo5nr957E56nTXE1"
            },
            {
                "id": "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-2",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC",
                "verificationMaterial": "z6MkjnP7TsFXr32rX3EoD6uUEzE2Fp2pPcGZYtBCEKL1fUAT"
            },
            {
                "id": "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-3",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC",
                "verificationMaterial": "z6MkofRWLT37dkx57TYhETh2ytBzifDMvV6dpVrLC5ugXm17"
            },
            {
                "id": "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-4",
                "verificationMethodType": "Ed25519VerificationKey2020",
                "controller": "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC",
                "verificationMaterial": "z6MkirAS5q6B4EQD1Vhe5vXFmorUpex6rLjuSH9eVYCHWK26"
            }
        ],
        "authentication": [
            "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-1",
            "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-2"
        ],
        "assertionMethod": [
            "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-3"
        ],
        "capabilityInvocation": [
            "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-4",
            "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-1"
        ],
        "capabilityDelegation": [
            "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-4",
            "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-2"
        ],
        "keyAgreement": [
            "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-1",
            "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-2",
            "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-3",
            "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#key-4"
        ],
        "service": [
            {
                "id": "did:cheqd:testnet:7Hty8uGafcEsU7EpJEAPUC#service-1",
                "serviceType": "type-1",
                "serviceEndpoint": [
                    "endpoint-1"
                ]
            }
        ],
        "alsoKnownAs": [
            "alias-1",
            "alias-2"
        ]
    },
    "metadata": {
        "created": "2021-01-01T00:00:00Z",
        "updated": "2021-01-01T00:00:00Z",
        "deactivated": false,
        "versionId": "e604fc38-a7b3-4bf7-b0b0-053ad9b15957",
        "nextVersionId": "",
        "previousVersionId": ""
    }
}